packages/backend/src/ws/WebSocketManager.ts

```diff
@@ -0,0 +1,111 @@
+import { Server, Socket } from 'socket.io';
+import http from 'http';
+import { normalizeEnsName } from '@dm3-org/dm3-lib-profile';
+import { checkToken } from '@dm3-org/dm3-lib-delivery';
+import { IDatabase } from '../persistence/getDatabase';
+import { ethers } from 'ethers';
+import { IWebSocketManager } from '@dm3-org/dm3-lib-shared';
+
+export const UNAUTHORIZED = 'unauthorized';
+export const AUTHORIZED = 'authorized';
+
+export class WebSocketManager implements IWebSocketManager {
+    private readonly connections: Map<string, Socket[]> = new Map();
```

Comment:
Yes, what is read-only is the pointer to the Map.  So it can't be resigned after the object was constructed. 
[This article gives a good explanation of why it's advantageous to use readonly/final](https://stackoverflow.com/a/154510)